samples/README.md

```diff
@@ -195,6 +195,21 @@ Documentation for the Search API For Shopping in
   </tr>
 </table>
 
+## ![](https://www.google.com/images/icons/product/compute_engine-32.png) Cloud Compute Engine API
+
+Lets you manage the Google Compute Engine instances in your projects.
+
+Documentation for the Cloud Compute Engine API in
+[PyDoc](https://cloud.google.com/compute/docs/tutorials/python-guide).
+
+<table>
+  <tr>
+    <td><a href="storage">samples/compute</a></td>
+    <td>Creates a Google Compute Engine instance to modify a samples image and upload the result
+        to a Cloud Storage bucket, then deletes the instance.</td>
+  </tr>
+</table>
```

Comment:
Will html code work in markdown file?

Reply:
As we can see [here](https://github.com/googleapis/google-api-python-client/tree/master/samples) it works just fine on GitHub :)